assets/sass/components/global/_googlesitekit-sharing-settings.scss

```diff
@@ -104,18 +104,7 @@
 
 	.googlesitekit-user-role-select--open {
 		align-items: flex-start;
-
-		.googlesitekit-user-role-select__button {
```

Comment:
As per the note about this under **Relevant technical choices** in the PR description, these CSS tweaks are needed to ensure the Close button flows after the roles in mobile viewports.